### ISSUE TITLE
Won't work in latest chrome.

### ISSUE THREAD

**Issue by rickrich** [rmerom/manual_geolocation#1]
1) manifest.json needs:
  "manifest_version": 2,

2) After that it loads but: 
Manual Geolocation 1.0.1
Override HTML5 GeoLocation API and return an arbitrary location.
Permissions  Reload (Ctrl+R)
ID: njdkecpknekialfnmfpmflikopfigopj
Loaded from: ~/manual_geolocation-master/manualgeolocation
Inspect views: background.html

Refused to execute inline event handler because it violates the following Content Security Policy directive: "script-src 'self' 'unsafe-eval'". Either the 'unsafe-inline' keyword, a hash ('sha256-...'), or a nonce ('nonce-...') is required to enable inline execution.

**Comment by rmerom** [comment]
Yes, this extension has not been maintained in the last couple of years.
Unfortunately I probably won't get around soon to fixing it, but I'd be
very happy if someone had the skills and the good will to go ahead and
update it.

**Comment by rickrich** [comment]
Too bad.  It was a nice addon.  Especially nice was the satellite (aerial)
views.  I wish it was in the Google Store!

On Sun, Mar 8, 2015 at 5:24 PM, rmerom notifications@github.com wrote:

> Yes, this extension has not been maintained in the last couple of years.
> Unfortunately I probably won't get around soon to fixing it, but I'd be
> very happy if someone had the skills and the good will to go ahead and
> update it.
> On Mar 7, 2015 7:00 PM, "rickrich" notifications@github.com wrote:
> 
> > 1) manifest.json needs:
> > "manifest_version": 2,
> > 
> > 2) After that it loads but:
> > Manual Geolocation 1.0.1
> > Override HTML5 GeoLocation API and return an arbitrary location.
> > Permissions Reload (Ctrl+R)
> > ID: njdkecpknekialfnmfpmflikopfigopj
> > Loaded from: ~/manual_geolocation-master/manualgeolocation
> > Inspect views: background.html
> > 
> > Refused to execute inline event handler because it violates the following
> > Content Security Policy directive: "script-src 'self' 'unsafe-eval'".
> > Either the 'unsafe-inline' keyword, a hash ('sha256-...'), or a nonce
> > ('nonce-...') is required to enable inline execution.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/rmerom/manual_geolocation/issues/1.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/rmerom/manual_geolocation/issues/1#issuecomment-77778452
> .

## 

Rick Richardson rickrich@gmail.com http://rickrich1.home.comcast.net/
Linux printer drivers for JBIG (GDI printers): Konica Minolta magicolor,
                          HP LaserJet, Samsung, Xerox, Lexmark, and Oki.
                                             http://foo2zjs.rkkda.com/
Linux tools for geocaching   http://geo.rkkda.com/
Gnome Manual Duplex         http://sourceforge.net/projects/g-manual-duplex/

**Comment by marcomilone** [comment]
Please fix it, was very useful
